app/views/users/show.html.erb

```diff
@@ -1 +1,18 @@
-<h1>This is a view to show a single user</h1>
\ No newline at end of file
+<%= render partial: "partials/user", locals: {user: @user} %>
+
+<section class="bio-section">
+  <h2>Bio</h2>
+  <p><%= @user.bio %></p>
+</section>
+
+<section class="posts-section">
+  <h2>Posts</h2>
+  <div class="posts-container">
+    <ul class="posts-list">
+    <% @posts.each do |post| %>
+      <%= render partial: "partials/post", locals: {post: post} %>
+    <% end %>
+    </ul>
+  </div>
+  <button>Show all posts</button>
```

Comment:
User should see all posts while clicking on `show all posts` on user's show page.